### PR TITLE
Creating page updates

### PR DESCRIPTION
**Pull request by kittarlin**
Should close issues 66 and 174.  Also sorts/organizes list

### CODE REVIEW

File: www/creating-zooming-images.html [hunk 1]
```diff
@@ -121,8 +138,9 @@ <h2>Creating Zooming Images</h2>
...
     <tr>
-        <td><a href="https://github.com/usnistgov/pyramidio">PyramidIO</a></td>
-        <td>Java (command line and library)</td>
-        <td><a href="/examples/tilesource-dzi/">DZI</a></td>
+        <td><a href="http://www.zoomify.com/photoshop.htm">Zoomify Plugin</a></td>
+        <td>Free proprietary export plugin for Photoshop</td>
+        <td><a href="/examples/tilesource-zoomify/">Zoomify</a></td>
     </tr>
 </table>
+<p>If none of these options meet your needs, you can create your own tile-generating tool. See <a href="https://github.com/openseadragon/openseadragon/wiki/The-DZI-File-Format">this page</a> for more information the DZI format.</p>
```
Comment (iangilman) [inline www/creating-zooming-images.html]: Should be "more information on the". Also, let's have a blank line above this to separate it from the table.

**Comment by iangilman** [comment]
...and don't forget to add HDMake. :)

**Comment by iangilman** [comment]
Note: looks like this also takes care of #178.

**Comment by kittarlin** [comment]
Last update should address items 178, 168, 162, 152, and 66

File: www/creating-zooming-images.html [hunk 2]
```diff
@@ -45,30 +38,25 @@ <h2>Creating Zooming Images</h2>
...
-        <td><a href="/examples/tilesource-tms/">TMS</a></td>
+        <td><a href="https://github.com/mulka/tiler">Gmap Uploader Tiler</a></td>
+        <td>C++</td>
+        <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
     <tr>
-        <td><a href="http://www.vips.ecs.soton.ac.uk/">VIPS</a></td>
-        <td>command line tool and library for a number of languages</td>
-        <td><a href="/examples/tilesource-dzi/">DZI</a> via the <a href="http://libvips.blogspot.co.uk/2013/03/making-deepzoom-zoomify-and-google-maps.html">dzsave</a> feature</td>
+        <td><a href="https://www.microsoft.com/en-us/download/details.aspx?id=52341">HDMake</a></td>
```
Comment (iangilman) [inline www/creating-zooming-images.html]: Looks like this should be "HD View Utilities" and the description should be something like "Photoshop plugin and command-line tool (Windows)".

File: www/creating-zooming-images.html [hunk 3]
```diff
@@ -121,8 +143,16 @@ <h2>Creating Zooming Images</h2>
...
     <tr>
-        <td><a href="https://github.com/usnistgov/pyramidio">PyramidIO</a></td>
-        <td>Java (command line and library)</td>
-        <td><a href="/examples/tilesource-dzi/">DZI</a></td>
+        <td><a href="http://www.zoomify.com/photoshop.htm">Zoomify Plugin</a></td>
+        <td>Free proprietary export plugin for Photoshop</td>
+        <td><a href="/examples/tilesource-zoomify/">Zoomify</a></td>
     </tr>
 </table>
+<p>If none of these options meet your needs, you can create your own tile-generating tool. See <a href="https://github.com/openseadragon/openseadragon/wiki/The-DZI-File-Format">this page</a> for more information on the DZI format.</p>
```
Comment (iangilman) [inline www/creating-zooming-images.html]: I'd still like a blank line above this.

File: www/creating-zooming-images.html [hunk 4]
```diff
@@ -121,8 +143,16 @@ <h2>Creating Zooming Images</h2>
...
+<h3>Hosting Zooming Images</h3>
```
Comment (iangilman) [inline www/creating-zooming-images.html]: Actually I think these should be integrated with the list above. They are both for IIIF, so they can go in that section. The description can be just "server" or something.

File: www/index.html [hunk 5]
```diff
@@ -172,6 +172,9 @@ <h3>Code Plugins</h3>
             <li><a href="https://github.com/thejohnhoffer/viaWebGL">OpenSeadragonGL</a> allows you to run WebGL shaders on all tiles in OpenSeadragon.</li>
             <li><a href="https://github.com/KTGLeiden/Openseadragon-screenshot">OpenSeadragonScreenshot</a> allows you to make a screenshot of your viewport, with optional magnification.</li>
             <li><a href="https://github.com/picturae/OpenSeadragonGuides">OpenSeadragonGuides</a> allows you to add horizontal and vertical guidelines to the Openseadragon viewer.</li>
+            <li><a href="https://github.com/thejohnhoffer/viaWebGL">GLSL Shaders on OpenSeadragon</a> allows you to add shaders to zoomable tiles or any other image viaWebGL.</li>
```
Comment (iangilman) [inline www/index.html]: Looks like this is already in the list (see line 172). Sorry for the state of our bug list!

File: www/creating-zooming-images.html [hunk 6]
```diff
@@ -98,7 +98,19 @@ <h2>Creating Zooming Images</h2>
         <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
     <tr>
-        <td colspan="3"><h4>IIIF</h4></td>
+        <td colspan="3"><h4>IIIF</h4>
+            (requires a IIIF-compliant image server with JPEG 2000 support, like <a href="https://github.com/pulibrary/loris">Loris</a>; see also <a href="http://iiif.io">iiif.io</a> for more details)
```
Comment (iangilman) [inline www/creating-zooming-images.html]: Actually this comment only applies to Kakadu; the other two are servers themselves.

**Comment by iangilman** [comment]
Deployed: 

http://openseadragon.github.io/examples/in-the-wild/

http://openseadragon.github.io/examples/creating-zooming-images/

**Pull request merged**